ui/app/components/layout/app.sidebar.tsx

```diff
@@ -200,11 +200,7 @@ export function AppSidebar({ ...props }: React.ComponentProps<typeof Sidebar>) {
       </SidebarContent>
       <SidebarFooter className="relative">
         {state === "expanded" && <TensorZeroStatusIndicator />}
-        <SidebarTrigger className="justify-left mt-1 flex">
-          <span className="sr-only">
-            {state === "expanded" ? "Collapse sidebar" : "Expand sidebar"}
```

Comment:
qq, is removing this `sr-only` element because we already have accessibility hints in sidebar.tsx:293 (the "toggle sidebar" span)?

Reply:
Yeah, these actually never get rendered at all because `SidebarTrigger` renders its own children, so this was essentially dead code.